roles/openstack-setup/tasks/users.yml

```diff
@@ -15,8 +15,8 @@
                     login_password={{ secrets.admin_password }}
                     insecure={{ insecure|default(omit) }}
   with_items: "{{ keystone.services }}"
-  when: 
-    - (item.name.enabled is undefined or item.name.enabled|bool)
+  when:
+    - ({{item.name}}.enabled is undefined or {{item.name}}.enabled|bool)
```

Comment:
This is a bit different, we don't always define the "enabled" for service endpoints (keystone, nova etc) so this will create those services as well as anything that has enabled defined.